extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,16 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_iframePing': {
+    'triggers': {
+      'pageLoad': {
+        'on': 'visible',
+        'request': 'iframeMessage',
+        'iframePing': true,
```

Comment:
Ah so that's why I was failing validation. I'll fix this with the requested rename once I have the needed guidance.